### DISCUSSION TITLE
prisma introspect genereate a file but I dont get a data model inside my Prisma schema.

### DISCUSSION THREAD

**Discussion by alan345** [prisma/prisma#2899] [Q&A] [answered]
`prisma introspect`
```
(node:19598) Warning: Accessing non-existent property 'count' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:19598) Warning: Accessing non-existent property 'findOne' of module exports inside circular dependency
(node:19598) Warning: Accessing non-existent property 'remove' of module exports inside circular dependency
(node:19598) Warning: Accessing non-existent property 'updateOne' of module exports inside circular dependency
Warning: Your Prisma server and Prisma CLI are currently out of sync. They should be on the same minor version.
  
Prisma CLI version: prisma/1.34.10 (darwin-x64) node-v14.4.0
Prisma server version: 1.8.4
  
For further information, please read: http://bit.ly/prisma-cli-server-sync
  
? What kind of database do you want to introspect? MySQL
? Enter database host localhost
? Enter database port 3306
? Enter database user root
? Enter database password [hidden]
Introspecting database nacho@prod 828ms
Created datamodel definition based on 130 database tables.
Created 1 new file:    GraphQL SDL-based datamodel (derived from existing database and the existing datamodel)

  datamodel-1593549930003.prisma

```
A new file `datamodel-1593549930003.prisma` has been created but file `schema.prisma` has not been updated with the prisma2 data model.

It seems to be a conflict with `prisma1`

**Comment by ryands17** [comment, marked as answer]
Hey @alan345 :wave: 
I think you are using a global version of Prisma. It would be best if you uninstall that and install `@prisma/cli` and `@prisma/client` locally. Then you can run `npx prisma introspect` to introspect your database.

**Reply by janpio** [reply]
More important: The global `prisma` being used is Prisma 1, not Prisma 2.

**Reply by alan345** [reply]
I see, I will `nom uninstall -g prisma` and `npm install @prisma/cli --save-dev` and always use `npx prisma ..`

**Comment by ludralph** [comment]
Hi @alan345,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
